src/index.js

```diff
@@ -200,6 +204,81 @@ class IpfsRepo {
   exists (callback) {
     this.version.exists(callback)
   }
+
+  /**
+   * Get repo status.
+   *
+   * @param {Object}  options
+   * @param {Boolean} options.human
+   * @param {function(Error, Object)} callback
+   * @return {void}
+   */
+  stat (options, callback) {
+    if (typeof options === 'function') {
+      callback = options
+      options = {}
+    }
+
+    options = Object.assign({}, {human: false}, options)
+
+    parallel([
+      (cb) => this.config.get('Datastore.StorageMax', (err, max) => {
+        if (err) {
+          cb(null, new Big(noLimit))
+        } else {
+          cb(null, new Big(max))
+        }
+      }),
+      (cb) => this.version.get(cb),
+      (cb) => this.blocks.query({}, (err, list) => {
+        list = list || []
+
+        const count = new Big(list.length)
+        let size = new Big(0)
+
+        list.forEach(block => {
+          size = size
+            .plus(block.value.byteLength)
+            .plus(block.key._buf.byteLength)
+        })
+
+        cb(err, {
+          count: count,
+          size: size
+        })
+      }),
+      (cb) => getSize(this.datastore, cb),
+      (cb) => getSize(this.keys, cb)
+    ], (err, results) => {
+      if (err) return callback(err)
+
+      let size = results[2].size
+        .plus(results[3])
+        .plus(results[4])
+
+      if (options.human) {
+        size = size.div(1048576)
```

Comment:
why not use something like https://www.npmjs.com/package/pretty-bytes instead?

Reply:
The spec says MiB so it must be in base 2 and not 10. Unless we could find another package. But why importing another package if we can just divide by that number?

Reply:
Ah okay, the package only makes sense if the output should be dynamically adjusted depending on the size.